manywheel/build_rocm.sh

```diff
@@ -59,22 +59,31 @@ elif [[ "$OS_NAME" == *"Ubuntu"* ]]; then
     LIBELF_PATH="/usr/lib/x86_64-linux-gnu/libelf.so.1"
 fi
 
+# rocm3.8 and later use TensileLibrary.dat
 if [[ $ROCM_VERSION == "rocm3.7" ]]; then
     TENSILE_LIBRARY_NAME=TensileLibrary.yaml
-elif [[ $ROCM_VERSION == "rocm3.8" ]]; then
+else
```

Comment:
```suggestion
elif [[ $ROCM_VERSION == "rocm3.8" || $ROCM_VERSION == "rocm3.9" ]]; then
```

Reply:
I did consider that, but IIUC the tensile library name extension will hereafter always be *.dat, so the rocm3.7 case was the only exceptional case.

Reply:
got it. thanks for the explanation